docs/content/preview/api/ysql/the-sql-language/statements/dml_insert.md

```diff
@@ -92,9 +122,34 @@ DO NOTHING | DO UPDATE SET *update_item* [ , ... ] [ WHERE *condition* ]
 
 #### *condition*
 
+An expression that returns a value of type boolean. 
+All rows are locked when the `ON CONFLICT DO UPDATE` action is taken, but only rows where this expression returns true get updated.
+
+This condition is evaluated after a conflict has been identified as a candidate to update.
+
+Only `NOT DEFERRABLE` constraints and unique indexes are supported as arbiters.
```

Comment:
Is this meant to be part of the *condition* docs or should it be a separate section? In PG docs indentation makes this difference clear -- not sure how it'd look in our docs yet.

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/20553/commits/de2847b2fbeea7d751cc6028c4f6abd675efd708